app/helpers/news_helper.rb

```diff
@@ -38,12 +24,15 @@ def serialize_news_items(news_items)
 
   def get_default_news_items(rewrite_cache: false)
     Rails.cache.fetch("default_news_items", force: rewrite_cache) do
-      distribute_reads(max_lag: ApplicationController::MAX_ACCEPTABLE_REPLICATION_LAG, lag_failover: true) do
-        news_items = default_news_query
-        news_items = backup_default_news_query if news_items.empty? || news_items.length < DEFAULT_NEWS_LIMIT
-
-        serialize_news_items(news_items)
-      end
+      news_items = NewsServices::RetrieveDefaultNewsItems.call.try(:result)
+      serialize_news_items(news_items) if news_items.present?
     end
   end
+
+  def get_news_items_with_votes(json_news_items)
```

Comment:
Can we just call this directly in the controller when needed instead of routing through the Helper?